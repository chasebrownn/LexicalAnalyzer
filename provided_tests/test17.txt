yDe3S (((F)*).((Q).(h)))|(k) #
" FFFFFQhk kQhFFF kFFFF "
