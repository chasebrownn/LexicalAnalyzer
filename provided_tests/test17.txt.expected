yDe3S , "FFFFFQh"
yDe3S , "k"
yDe3S , "k"
yDe3S , "Qh"
ERROR
